pallets/parachain-staking/src/lib.rs

```diff
@@ -1757,6 +1767,13 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> Result<(), &'static str> {
+			crate::try_state::validate_candiate_pool::<T>()?;
```

Comment:
I would rather like to see a single `try_state()` function exposed, which then calls the other auxiliary functions. It's less burden for someone that is going over this pallet's code to see a single call in here.